orchestra/analytics/latency.py

```diff
@@ -55,12 +55,17 @@ def work_time_df(projects, human_only=True, complete_tasks_only=True):
             get_project_information(project.id),
             human_only, complete_tasks_only)
         for project in projects))
-    df = DataFrame(time_data)
+
+    # Ensure that even an empty DataFrame has the correct columns
+    columns = ['assignment_level', 'calendar_time', 'end_datetime',
+               'iteration', 'project_description', 'project_id',
+               'start_datetime', 'task_id']
```

Comment:
are these names spelled out elsewhere ever? if so, should they be a module level var `DATAFRAME_COLUMNS` somewhere?